app/src/main/java/com/duckduckgo/app/statistics/store/PixelDao.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2020 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.statistics.store
+
+import androidx.room.*
+import com.duckduckgo.app.statistics.model.PixelEntity
+import io.reactivex.Observable
+
+@Dao
+interface PixelDao {
+
+    @Query("select * from pixel_store")
+    fun unsentPixels(): Observable<List<PixelEntity>>
```

Comment:
nit: this is actually getting all pixels in the DB so I would just call it `pixels`. The name `unsentPixels` seems to hint that there's another kind of pixels in the DB, while it is not.
What do you think?

Reply:
You're technically right @aitorvs, although for some reason I do get some sort of reassurance from the name `unsentPixels` that I don't get from `pixels` (as it could read like we save all of them forever without it)

maybe `pendingPixels()` would be clear and still reassuring? (I don't feel strongly about this)